mouadd9/student-documents-request-frontend-Angular
# fixed style

used proper styling , steps to test : 
- go fill in the db.json 
- go to the demandes section 
- observe how styles are made 
- reject or approve demandes 
- filter out demands according to type
